### ISSUE TITLE
NaN prizes and edge weights

### ISSUE THREAD

**Issue by agitter** [fraenkel-lab/OmicsIntegrator#26]
`forest.py` should check for NaN prizes and edge weights.  I accidentally ran it with a NaN prize.  It propagates the data to msgsteiner, producing output files that are technically valid but don't make sense.  In my case, they ignored the specified root node.

I'll make this fix myself when time permits.

**Comment by agitter** [comment]
And it should also check for infinite prizes, which cause similar problems.

**Comment by agitter** [comment]
Does anyone else have an opinion about the preferred `forest.py` behavior when we encounter NaN or infinite prizes?  We could:

1) Add a new warning that states the number of invalid prizes and continue running with the valid prizes
2) Stop execution

I will implement 1. unless there are other opinions.  It seems most consistent with the rest of our error handling.

**Comment by AmandaKedaigle** [comment]
Yeah, I think 1 is good. There could be some kind of threshold for what percentage of prizes must be valid for it to continue, the way we handle protein name mismatches, but I don't feel strongly either way.
